### ISSUE TITLE
Unable to get to connection fields with keyboard

### ISSUE THREAD

**Issue by MikhailArkhipov** [Microsoft/RTVS#3172]
Click edit connection button
Try getting to edit fields with keyboard
You can't

Also, ESC does not act as Cancel, editing does not close

**Comment by AlexanderSher** [comment]
Where should focus on cancel go? Back to the edit button?

**Comment by MikhailArkhipov** [comment]
Whatever the `Cancel` button does :-)
